app/routes/board_routes.py

```diff
@@ -11,3 +11,17 @@ def get_all_boards():
     boards = Board.query.all()
     boards_response = [board.to_dict() for board in boards]
     return make_response(jsonify(boards_response), 200)
+
+@board_bp.route("", methods=["POST"])
+def create_board():
+    request_body = request.get_json()
+
+    if "title" not in request_body or "owner" not in request_body:
+        abort(make_response(jsonify({"details": "Invalid data"}), 400))
+
+    new_board = Board(title=request_body["title"],
+                      owner=request_body["owner"])
+    db.session.add(new_board)
+    db.session.commit()
+
+    return jsonify(new_board.to_dict()), 201
```

Comment:
I think we probably want to keep our returns uniform, so you might want to add a make_response() to wrap around the jsonify.